contentwarehouse/snippets/search_documents_sample.py

```diff
@@ -53,9 +52,12 @@ def search_documents_sample(
         histogram_query='count("DocumentSchemaId")'
     )
 
+    request_metadata = {"user_info": {"id": user_id}}
```

Comment:
Can you change this to create an instance of the [`RequestMetadata`][1] class instead of a generic dictionary?

https://cloud.google.com/python/docs/reference/contentwarehouse/latest/google.cloud.contentwarehouse_v1.types.RequestMetadata